Example/iOSDFULibrary/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```

Comment:
Add `NSPrivacyAccessedAPIType` with type "User defaults APIs" and reason `CA92.1`.

Reply:
I don't see UserDefaults used by DFU Example App. It's used by McuManager Example app. But if you want me to, I'll add it.

Reply:
As I said, the app is using `@AppStorage` properties.